pages/common/licensor.md

```diff
@@ -0,0 +1,28 @@
+# licensor
+
+> Write licenses to stdout.
+> More information: <https://github.com/raftario/licensor>.
+
+- Write the MIT license to a file named `LICENSE`:
+
+`licensor {{MIT}} > {{LICENSE}}`
+
+- Write the MIT license with a placeholder copyright notice to a file named `LICENSE`:
+
+`licensor -p {{MIT}} > {{LICENSE}}`
+
+- Specify a copyright holder named Bobby Tables:
+
+`licensor {{MIT}} {{"Bobby Tables"}} > {{LICENSE}}`
+
+- Specify licence exceptions with a WITH expression:
+
+`licensor {{"Apache-2.0 WITH LLVM-exception"}} > {{LICENSE}}`
```

Comment:
```suggestion
`licensor "{{Apache-2.0 WITH LLVM-exception}}" > {{LICENSE}}`
```

Just a suggestion. 😄

Reply:
Technically, the quotation marks are only required if the license name contains spaces, so I feel like the quotes make more sense to be placed inside of the curly brackets.

However, any expression that has a "WITH" statement will always have spaces in it, so I do see the rational for this change.

And of course, if this change is the recommended style, I can make this change.

Reply:
It's like a convention. If there is no something wrong, we prefer `"{{placeholder}}"` rather than `{{"placeholder"}}`.

Reply:
@blueskyson So, to clarify, would you prefer that this change is made?

Reply:
@superatomic yeah please, apply the change.